packages/chrome.extensions.vm/tools/chocolateyinstall.ps1

```diff
@@ -0,0 +1,91 @@
+$ErrorActionPreference = 'Stop'
+Import-Module vm.common -Force -DisableNameChecking
+
+try {
+  $extensions = @(
+    # MetaMask
+    'nkbihfbeogaeaoehlefnkodbefgpgknn'
+    # Phantom
+    'bfnaelmomeimhlpmgjnjophhpkkoljpa'
+    # BNB Chain Wallet
+    'fhbohimaelbohpjbbldcngcnapndodjp'
+    # Avira Password Manager
+    'caljgklbbfbcjjanaijlacgncafpegll'
+    # KeePassXC-Browser
+    'oboonakemofpalcgghocfoadofidjkkk'
+    # Yoroi
+    'ffnbelfdoeiohenkjibnmadjiehjhajb'
+    # XDEFI Wallet
+    'hmeobnfnfcmdkdcmlblgagmfpfboieaf'
+    # ...
+  )
+
+  # Installing the extensions under `ExtensionInstallForcelist` so
+  #      it can be installed and enabled silently, without user interaction.
+  # By default, this registry key does not exist and it is not used within Flare-VM.
+  # Ref: https://chromeenterprise.google/policies/?policy=ExtensionInstallForcelist
+
+  $regKeyPath = "HKLM:\SOFTWARE\WOW6432Node\Policies\Google\Chrome\ExtensionInstallForcelist"
+  $updateUrl = "https://clients2.google.com/service/update2/crx"
+
+  New-Item -Path $regKeyPath -Force -ea 0 | Out-Null
+  $valueName = 1
+  Foreach ($extensionId in $extensions)
+  {
+    New-ItemProperty -Path "$regKeyPath" -Name "$valueName" -Type String -Value "$extensionId;$updateUrl" -Force -ea 0 | Out-Null
+    $valueName += 1
+  }
+
+  $maximumTries = 5
+  $chromePath = "${Env:ProgramFiles}\Google\Chrome\Application\chrome.exe"
+  $extensionsDir = "${Env:LocalAppData}\Google\Chrome\User Data\Default\Extensions"
+
+  # Stop Chrome if it is already running.
+  Stop-Process -Force -Name Chrome -ea 0
+
+  # Start Chrome to load the extensions.
+  $chromeProcess = Start-Process -FilePath $chromePath -passthru
+
+  $tries = 0
+  $loaded = $false
+  while ((-not $loaded) -and ($tries -ne $maximumTries))
+  {
+    # Wait for the extensions to be loaded.
+    Start-Sleep -Seconds 60
```

Comment:
[nitpick] can we lower the time a bit (and increase $maximumTries)? I had to wait a bit until the extensions were installed and I think we could save some time here.
```suggestion
    Start-Sleep -Seconds 30
```

Reply:
Done.